usaspending_api/awards/serializers.py

```diff
@@ -13,6 +13,8 @@
 class AwardTypeAwardSpendingSerializer(serializers.Serializer):
```

Comment:
If we're moving to a `views_v2` directory, we should also move this to a `serializers_v2` directory.